practice/1_python_part_1/task6.py

```diff
@@ -20,5 +20,9 @@
 
 
 def get_min_max(filename: str) -> Tuple[int, int]:
-    ...
-
+    int_list = []
+    with open(filename) as opened_file:
+        for line in opened_file:
+            int_list.append(int(line))
+        output_tuple = tuple((min(int_list), max(int_list)))
```

Comment:
no need for `tuple(...)`

Reply:
output changed to `return min(int_list), max(int_list)`